assets/src/scripts/components/graph/view/construction.js

```diff
@@ -138,8 +190,18 @@ export default function (elem, {elements, cursorWaterLevel}, store, opts, contai
     // Remove any previously drawn children
     container.selectAll('*').remove();
 
+    // Draw well raw borehole element
+    const borehole = drawWellBorehole(store, container, opts, elements);
+    let waterLevelElements = elements;
+    if (!elements || elements.length == 0) {
+        waterLevelElements = [borehole];
+        if (cursorWaterLevel) {
+            cursorWaterLevel.height = borehole.left.y2 - cursorWaterLevel.y;
+        }
+    }
```

Comment:
when there are no elements it is actually null and current water level object does not have a water height.